feature/src/main/resources/feature.xml

```diff
@@ -320,11 +320,6 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.provider/glesys/${jclouds.version}</bundle>
     </feature>
 
-    <feature name='jclouds-hpcloud-compute' description='HP Cloud Compute' version='${project.version}' resolver='(obr)'>
-        <feature version='${project.version}'>jclouds-api-openstack-nova</feature>
-        <bundle>mvn:org.apache.jclouds.provider/hpcloud-compute/${jclouds.version}</bundle>
-    </feature>
```

Comment:
Yeah. I'll split this into 2 PRs: the first one to fix the build (ssh deps and provider removal), and another one to add the new providers.
The PR build now complains about a google dependency config, and that shouldn't block the ssh fix. Let's properly scope each PR.